luarules/gadgets/cmd_mex_denier.lua

```diff
@@ -11,6 +11,16 @@ function gadget:GetInfo()
 	}
 end
 
+-- Some of these maps have more than 2 metal spots, disable mex denier
```

Comment:
ZK has a limit of 5 (i.e. it takes 6 or more not to become automatically disabled). Historically that was enough to cover all the speedmetal variantes without excluding any real map.

Reply:
Cloud9_V2 has 6 :-(